test/acceptance/features/ApplyLandFunding.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Apply for land funding
```

Comment:
As part of development we follow standardJS to make sure consistency across repos. So file names need to contain only lower case and words need separating with a hyphen (-)

Could we re-name to apply-land-fund.feature